src/components/Calculator.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import Result from './Result';
+
+const Calculator = () => {
+  const handleClick = (buttonName) => (`${buttonName}clicked`);
```

Comment:
[OPTIONAL]
- [ ] I noticed this `handleClick` function, but it appears to be doing nothing. As a best practice and in line with the YAGNI, I would recommend that you remove it from your code, you can always add it back at the time that you need it.